src/domain/synthetics/stats/useFeesInfo.ts

```diff
@@ -0,0 +1,104 @@
+import { gql } from "@apollo/client";
+import { useFeesSummary } from "domain/stats";
+import { BigNumber } from "ethers";
+import { getSyntheticsGraphClient } from "lib/subgraph";
+import useSWR from "swr";
+
+const totalFeeQuery = gql`
+  query totalFeesInfo {
+    position: positionFeesInfoWithPeriod(id: "total") {
+      totalBorrowingFeeUsd
+      totalPositionFeeUsd
+    }
+    swap: swapFeesInfoWithPeriod(id: "total") {
+      totalFeeReceiverUsd
+      totalFeeUsdForPool
+    }
+  }
+`;
+
+const weeklyFeeQuery = gql`
+  query weeklyFeesInfo($lastTimestamp: Int!) {
+    position: positionFeesInfoWithPeriods(where: { id_gte: $lastTimestamp, period: "1d" }) {
+      totalBorrowingFeeUsd
+      totalPositionFeeUsd
+    }
+
+    swap: swapFeesInfoWithPeriods(where: { id_gte: $lastTimestamp, period: "1d" }) {
+      totalFeeReceiverUsd
+      totalFeeUsdForPool
+    }
+  }
+`;
+
+export default function useFeesInfo(chainId: number) {
+  const { data: feesSummaryByChain } = useFeesSummary();
+  const lastUpdatedAt = feesSummaryByChain[chainId]?.lastUpdatedAt;
+
+  async function fetchFeesInfo(chainId: number) {
+    try {
+      const client = getSyntheticsGraphClient(chainId);
+      const { data: weeklyFeesInfo } = await client!.query({
+        query: weeklyFeeQuery,
+        variables: {
+          lastTimestamp: lastUpdatedAt,
```

Comment:
it still won't work properly because fetchFeesInfo don't accept lastUpdatedAt in params and get it from external scope at the render